shared/base/view.js

```diff
@@ -195,7 +195,7 @@ module.exports = BaseView = Backbone.View.extend({
    * Get the HTML for the view, including the wrapper element.
    */
   getHtml: function() {
-    var html = this.getInnerHtml(),
+    var html = this.options.lazy ? '' : this.getInnerHtml(),
```

Comment:
Seems like we should just not be calling `getHtml` if it's a lazy view, instead we should verify why `rendr-handlebars` is trying to even render it on the server... i used to have a fix for this problem on my old computer... :'(